README.md

```diff
@@ -11,19 +11,16 @@ designed with unique features that help with more specific use cases.
 Pebble is organized as a single binary that works as a daemon and also as a
 client to itself. When the daemon runs it loads its own configuration from the
 _$PEBBLE_ directory, as defined in the environment, and also writes down in
-that same directory its state and unix sockets for communication.
+that same directory its state and unix sockets for communication. If that variable
+is not defined, Pebble will attempt to look for its configuration from a default
+system-level setup at _/var/lib/pebble/default_. Using that directory is encouraged
```

Comment:
Nit: For path names, would `<code>` spans (via backticks) be more typical / readable than italics (via underscores)? For env vars like `$PEBBLE`, seems like you've mixed underscores and backticks -- would be good to be consistent.

Reply:
Thanks for raising that. I've moved all cases (hopefully) of code, paths, and environment variables to backticks.